ckanext/activity/model/activity.py

```diff
@@ -226,13 +226,33 @@ def _activities_limit(
     return q
 
 
+def _activities_defer_data():
+    """
+    This function is used to defer the bulk Data Coloum from activity lists.
```

Comment:
"Coloum" -> "column"

Might also be worth mentioning what 'defer' means in this context, eg "This function is used to avoid unnecessarily loading the bulk Data column"